plugins/inventory/xen_orchestra.py

```diff
@@ -138,21 +139,41 @@ def create_connection(self, xoa_api_host):
         self.conn = create_connection(
             '{0}://{1}/api/'.format(proto, xoa_api_host), sslopt=sslopt)
 
+    CALL_TIMEOUT = 100
+    """Number of 1/10ths of a second to wait before method call times out."""
+    def call(self, method: str, params: dict) -> dict:
```

Comment:
Please add a newline and remove the typing hints, they won't work with Python 2.7:
```suggestion

    def call(self, method, params) -> dict:
```